src/clients/near-wallet-selector.ts

```diff
@@ -256,7 +260,19 @@ export class NearWalletSelectorBridgeClient {
       prover_args: proverArgsSerialized,
     }
     const serializedArgs = BindTokenArgsSchema.serialize(args)
+
     const wallet = await this.selector.wallet()
+    const accounts = await wallet.getAccounts()
+    const accountId = accounts[0].accountId
+
+    // Retrieve required deposit dynamically for bind_token
+    const bindDepositStr = await this.viewFunction({
+      contractId: this.lockerAddress,
+      methodName: "required_balance_for_bind_token",
+      args: { account_id: accountId },
```

Comment:
I don't see `account_id` arg for this function https://github.com/Near-One/omni-bridge/blob/main/near/omni-bridge/src/storage.rs#L195